.github/workflows/publish-release.yml

```
@@ -0,0 +1,59 @@
+name: Publish release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version number to tag and release'
+        required: true
+        type: string
+      prerelease:
+        description: 'Release as pre-release'
+        required: false
+        type: boolean
+        default: false
+
+jobs:
```

```suggestion
permissions:
  contents: read

jobs:
```

See https://github.com/fastify/fastify/pull/6035